.github/workflows/TitleVerifier.yaml

```diff
@@ -0,0 +1,17 @@
+name: Title Verifier
+
+on:
+  pull_request_target:
+    types: [push, opened, edited, reopened]
+
```

Comment:
Could you please rename the file for we do not have capitalized letters? 
For example, could it be title-verify.yml

Reply:
done